code/controllers/subsystem/garbage.dm

```diff
@@ -160,8 +167,14 @@ SUBSYSTEM_DEF(garbage)
 		fail_counts[level]++
 		switch (level)
 			if (GC_QUEUE_CHECK)
+				#ifdef TESTING
+				if(reference_find_on_fail[refID])
+					D.find_references()
 				#ifdef GC_FAILURE_HARD_LOOKUP
```

Comment:
Might as well remove this define then

Reply:
@Cyberboss that define is for looking up on every single gc failure, the one before it is user-specifiable